### PR TITLE
Add CI installation tests for Windows

### PR DESCRIPTION
**Pull request by DDtKey**
This pull request created for publication after [2.1.0](https://github.com/dotenv-linter/dotenv-linter/issues/217), because the Windows version will become available. As discussed earlier in #220 ([here](https://github.com/dotenv-linter/dotenv-linter/pull/220#issuecomment-653105102)).

After the release of **2.1.0**, I will add changes to the `CHANGELOG.md` and the _tests will have to pass successfully_

#### ✔ Checklist:

- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [X] Tests for the changes have been added (for bug fixes / features);

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -273,6 +273,19 @@ jobs:
...
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup dotenv-linter
+        shell: bash # it uses Git Bash, i.e. MinGW
+        run: |
+          mkdir -p $HOME/bin
+          sh "${GITHUB_WORKSPACE}/install.sh" -b $HOME/bin
+      - name: Verify successful installation
+        run: $HOME/bin/dotenv-linter -v
```
Comment (mgrachev) [inline .github/workflows/ci.yml:287]:
```suggestion
        run: $HOME/bin/dotenv-linter.exe -v
```
I suppose you forgot the `.exe` extension 🤔

Comment (DDtKey) [reply .github/workflows/ci.yml:287]: Yeah, thanks :)

**Comment by mgrachev** [comment]
@DDtKey Please look at the output:
```
dotenv-linter installer: The latest version will be installed.
D:\a\dotenv-linter\dotenv-linter/install.sh: line 85: required_cmd: command not found
Archive:  /tmp/tmp.R8OerubWD6/dotenv-linter-win-x64.zip
  inflating: /tmp/tmp.R8OerubWD6/dotenv-linter.exe  
dotenv-linter installer: Successfully installed dotenv-linter 2.1.0 to /c/Users/runneradmin/bin/dotenv-linter.exe
```

**Comment by DDtKey** [comment]
> @DDtKey Please look at the output:
> 
> ```
> dotenv-linter installer: The latest version will be installed.
> D:\a\dotenv-linter\dotenv-linter/install.sh: line 85: required_cmd: command not found
> Archive:  /tmp/tmp.R8OerubWD6/dotenv-linter-win-x64.zip
>   inflating: /tmp/tmp.R8OerubWD6/dotenv-linter.exe  
> dotenv-linter installer: Successfully installed dotenv-linter 2.1.0 to /c/Users/runneradmin/bin/dotenv-linter.exe
> ```

Oh, good catch! :+1:
 85 line is `required_cmd unzip`when has typo: require**d**_cmd
Therefore, the command was simply ignored, because unzip is actually installed.

I fixed the script! Thank you!

**Comment by mgrachev** [comment]
@DDtKey Thanks! 👍

**Pull request merged**